cypress/integration/barong_login_specs.js

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
can we change default cypress folder to spec/barong_login.specs.js

Reply:
You mean change ./cypress/integration/ to ./spec/? Ok, and what with fixtures and support folders?